modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -265,9 +265,9 @@ export interface HalfSignedAccountTransaction {
 export interface HalfSignedTransaction extends HalfSignedAccountTransaction {
   halfSigned: {
     recipients: Recipient[];
-    expireTime: number;
-    contractSequenceId: number;
-    sequenceId: number;
+    expireTime?: number;
```

Comment:
If pulling from abstract-eth, you want have to affect the core logic.

Reply:
Actually will still have to update this because signTransaction method in the base class Eth is actually using HalfSignedTransaction from sdk-core. So it is giving me error 

expireTime, contractSequenceId, sequenceId fields are missing from the type. These fields are not required in abstract-eth or polygon. So have made this change

Reply:
You can use `Partial<>` to extend and make optional required fields instead of changing the original intent.

Reply:
Have removed the changes in the sdk-core